nautobot/utilities/utils.py

```diff
@@ -386,20 +386,5 @@ def get_filterset_for_model(model):
     return None
 
 
-def is_truthy(arg):
```

Comment:
Possibly add `from nautobot.core.settings_funcs import is_truthy` to this file so that in case anyone (e.g. plugin or Job) is importing from this file, it'll still work?

Reply:
I believe `is_truthy` was added here in an initial attempt to remove it from nautobot_config.py as this would cause circular dependencies it was moved to settings_funcs I don't think it was here long enough to warrant the addition for backwards compatability.

Reply:
Seconded. We want it to be in `nautobot.core.settings_funcs` so it is isolated, whereas `nautobot.utilities.utils` is rife with inter-dependencies and could result in circular imports.